examples/diffusers/quantization/quantize.py

```diff
@@ -219,6 +219,104 @@ def parse_args() -> argparse.Namespace:
 
     return parser.parse_args()
 
+def quantize_diffusion_model(pipe: DiffusionPipeline,
+                             backbone: torch.nn.Module,
```

Comment:
@peri044 do we need the module itself? cant we just do the pass by attr name thing and do getattr, setattr? since there's another name based argument as well?